transports/transport-rest/transport-rest-servlet/src/main/java/io/servicecomb/transport/rest/servlet/RestServletProducerInvocation.java

```diff
@@ -17,20 +17,17 @@
 package io.servicecomb.transport.rest.servlet;
 
 import io.servicecomb.common.rest.RestProducerInvocation;
-import io.servicecomb.common.rest.definition.RestOperationMeta;
 import io.servicecomb.common.rest.filter.HttpServerFilter;
 import io.servicecomb.core.definition.OperationMeta;
 import io.servicecomb.foundation.vertx.http.StandardHttpServletRequestEx;
 
 public class RestServletProducerInvocation extends RestProducerInvocation {
   @Override
-  protected RestOperationMeta findRestOperation() {
-    RestOperationMeta restOperationMeta = super.findRestOperation();
+  protected void findRestOperation() {
+    super.findRestOperation();
 
     boolean cacheRequest = collectCacheRequest(restOperationMeta.getOperationMeta());
```

Comment:
this variable is useless.

Reply:
which one?
cacheRequst?   it's setted to requestEx,  if copy request body controlled by this avriable.

Reply:
I missed the second line , it makes sense now.